source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -183,7 +183,7 @@ user are contained in the `credentialSubject` JSON object.
 
 ### Check your user’s identity credential matches the level of confidence needed
 
-You must check the `vot` claim in the JWT matches or exceeds the level of confidence you need to let your user access
+You must look for the `vot` claim in the ID Token then make sure the expected and achieved level of protection matches or exceeds the level of confidence you need to let your user access
```

Comment:
```suggestion
You must look for the `vot` claim in the ID token and make sure it is the right level for your service. This means
the level of protection must match or exceed the level of confidence a user needs to access 
```
The original sentence is a bit too long so I've tried to break it up - what do you think?

We might also need to be clearer about what it means for a level of protection to 'match' a level of confidence. Does this mean a `cl` level of protection would match a `P1` level of confidence or something else? What does `P0` match to (questions for @pauldougan or @philf999 probably)? It might be useful to put an example here so users don't make assumptions about what 'matches' means!

Reply:
The vot claim in the ID token only includes the authentication protection level not the identity confidence level.

I don't think this correctly explains the meaning of the claim or what the RP needs to check.